source/slang/emit.cpp

```diff
@@ -5951,8 +5951,28 @@ struct EmitVisitor
         // TODO: we should require either the extension or the version...
         requireGLSLVersion(430);
 
-        emit("layout(std430) buffer ");
+        Emit("layout(std430");
```

Comment:
Minor note: it is legal in GLSL to have multiple `layout` qualifiers on one declaration. While generating all the stuff as a single `layout` may lead to cleaner-looking output, it sometimes leads to more corner cases in the emit logic since we have to keep track of when a comma is needed, etc. (imagine if we had two different cases of optional layout stuff that could follow `std430`).

I don't think it is worth changing in this case, but we might want to have a policy of always emitting one `layout` modifier per line, with the small exception of keeping `binding` and `set` together.